src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -480,7 +480,7 @@ public function __toString()
         try {
             $content = $this->getContent();
         } catch (\LogicException $e) {
-            trigger_error($e->getMessage(), E_USER_ERROR);
+            return trigger_error($e, E_USER_ERROR);
```

Comment:
This is always the last line (see https://github.com/symfony/symfony/pull/15077#issuecomment-114484146)